style.css

```diff
@@ -0,0 +1,143 @@
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.logo {
+  margin-right: auto;
+  align-items: center;
+  color: #6070ff;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.social {
+  list-style-type: none;
+  display: inline-block;
+  width: 100%;
+}
+
+nav li {
+  list-style: none;
+  display: inline-block;
+  padding: 0 10px;
+  color: #344563;
+}
+
+.social li {
+  display: inline;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  margin-left: 100px;
+}
```

Comment:
- [ ] Lastly, as the previous reviewer mentioned, this `margin -left here is giving your project a look that is different from that of the Figma design it is also making your project overflow and I believe removing it should make your design looks as it should.  All you'll have to add is the right styles as shown in the Figma Design :100: 
- This is how your project currently looks.
[Screencast from 02-18-2023 11:40:00 AM.webm](https://user-images.githubusercontent.com/102699958/219861662-e7fdb0f0-75fc-4567-bb15-cadf10f1f176.webm)
- Figma Design
![image](https://user-images.githubusercontent.com/102699958/219862918-d6fa25f6-3259-4491-a24b-f167ea1edcbd.png)
- [ ] From the image above, you can see that you're also missing the background image. Kindly fix add that as well :wink: